src/locales/en.json

```diff
@@ -333,10 +333,10 @@
     },
     "lengths": {
       "title": "Durations",
```

Comment:
For consistency, I think we should use `Duration`, without the `s`. All the other categories are written in a singular form.

Reply:
Great catch, @panchovm !